examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>Could you add the math for say 2nd order ADER?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>